fuse_models/CHANGELOG.rst

```diff
@@ -0,0 +1,81 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package fuse_models
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.2.0 (2019-07-12)
+------------------
+* Get predict_to_current_time ROS param (`#17 <https://github.com/locusrobotics/fuse_rl/issues/17>`_)
```

Comment:
I left the changelog links pointing to the `fuse_rl` repo so that those PRs can still be viewed.